shell/services/node/node_service.cc

```diff
@@ -127,6 +128,12 @@ void NodeService::Initialize(
 
   node_bindings_->Initialize(isolate, isolate->GetCurrentContext());
 
+  if (!network_change_notifier_) {
+    network_change_notifier_ = net::NetworkChangeNotifier::CreateIfNeeded(
+        net::NetworkChangeNotifier::CONNECTION_UNKNOWN,
+        net::NetworkChangeNotifier::ConnectionSubtype::SUBTYPE_UNKNOWN);
+  }
```

Comment:
can be, NodeService::Initialize is not re-entrant

```suggestion
    network_change_notifier_ = net::NetworkChangeNotifier::CreateIfNeeded(
        net::NetworkChangeNotifier::CONNECTION_UNKNOWN,
        net::NetworkChangeNotifier::ConnectionSubtype::SUBTYPE_UNKNOWN);
```